conftest.py

```diff
@@ -82,3 +90,13 @@ def pytest_configure(config):
             }],
             ALLOWED_HOSTS=['*'],
         )
+
+
+
+for transport in [EventletHTTPTransport,
```

Comment:
Yeah we'd have to update these tests to pass it in directly, but I'd be ok with that in order to remove the legacy compatibility.